config/index.js

```diff
@@ -5,67 +5,152 @@ module.exports = {
   },
   rules: {
     'accessor-pairs': 'error',
+    'array-bracket-newline': 'off',
+    'array-bracket-spacing': ['error', 'never'],
+    'array-callback-return': 'error',
+    'array-element-newline': 'off',
+    'arrow-body-style': 'off',
+    'arrow-parens': 'error',
     'arrow-spacing': ['error', { 'before': true, 'after': true }],
+    'block-scoped-var': 'error',
     'block-spacing': ['error', 'always'],
     'brace-style': 'error',
+    'callback-return': 'error',
     'camelcase': ['error', { 'properties': 'never', 'allow': ['^UNSAFE_'] }],
+    'capitalized-comments': 'off',
+    'class-methods-use-this': 'off',
     'comma-dangle': ['error', 'always-multiline'],
     'comma-spacing': ['error', { 'before': false, 'after': true }],
     'comma-style': ['error', 'last'],
+    'complexity': 'off',
+    'computed-property-spacing': 'error',
+    'consistent-return': 'error',
+    'consistent-this': 'error',
     'constructor-super': 'error',
     'curly': 'error',
     'default-case': 'error',
+    'default-param-last': 'error',
     'dot-location': ['error', 'property'],
+    'dot-notation': 'error',
     'eol-last': 'error',
     'eqeqeq': ['error', 'allow-null'],
+    'for-direction': 'error',
+    'func-call-spacing': 'error',
+    'func-name-matching': 'error',
+    'func-names': 'off',
+    'func-style': 'off',
```

Comment:
Side-note: it would be nice at some point to turn `func-names` and `func-style` on. I like `func-names`, and I would like to adopt a common pattern for `func-style`.

Reply:
Yeah I had started with all the errors and then disabled the ones that were prohibitively noisy, since I do want to introduce this to the extension soonish. Let's open an issue for this though so that we remember.

Reply:
Created #21 and #22